x/wasm/types/types.go

```diff
@@ -256,6 +256,10 @@ func NewEnv(ctx sdk.Context, contractAddr sdk.AccAddress) wasmvmtypes.Env {
 	if nano < 1 {
 		panic("Block (unix) time must never be empty or negative ")
 	}
+	txCounter := TXCounter(ctx)
+	// todo: where to set the counter
+	_ = txCounter
```

Comment:
https://github.com/CosmWasm/cosmwasm/pull/1077/files#diff-9818bb419db10c5127f7290971c04fdaa13ae8aa7495f9d900ee688e33d15d4aR25

Reply:
This will be available in 1.0.0-beta
(We can pre-release test with 1.0.0-soon, but let's put this merge on hold until we are ready to move wasmd support to that.. or start a branch for 1.0.0 compatibility)